packages/oui-datagrid/src/datagrid.controller.js

```diff
@@ -50,11 +36,14 @@ export default class DatagridController {
 
             const builtColumns = this.ouiDatagridColumnBuilder.build(columnElements, this.$scope);
             this.columns = builtColumns.columns;
-            this.currentSorting = builtColumns.currentSorting;
 
-            // Once columns has been processed,
-            // we can proceed with the first loading
-            this.initPage();
+            if (this.rowsLoader || this.rowLoader) {
```

Comment:
`rowLoader` is only used to retrieve a complete single, row. This doesn't systematically use a remote paging.
=> `loadRowsData` should be able to called with a local pagination.

Reply:
Nice catch, it is now fixed!